concepticondata/conceptlists/Brid-2022-324.tsv

```diff
@@ -0,0 +1,325 @@
+Brid-2022-324-93	93	sky	1732	SKY	cielo					
+Brid-2022-324-94	94	fingernail	1258	FINGERNAIL	uña					
+Brid-2022-324-95	95	claw	72	CLAW	garra					
+Brid-2022-324-96	96	fin (dorsal)	213	FIN	aleta					
+Brid-2022-324-97	97	fish	227	FISH	pez, pescado					
+Brid-2022-324-98	98	wing	1257	WING	ala					
+Brid-2022-324-99	99	palm of hand	1183	PALM OF HAND	palma de la mano					
+Brid-2022-324-100	100	small, little	1246	SMALL	chico, pequeño					
+Brid-2022-324-101	101	child	1304	CHILD (YOUNG HUMAN)	niño					
+Brid-2022-324-102	102	horn	1393	HORN (ANATOMY)	cuerno					
+Brid-2022-324-103	103	club	1763	CLUB	macana, garrote					
+Brid-2022-324-104	104	soul, spirit	811	SOUL	alma					
+Brid-2022-324-105	105	shade, shadow	1388	SHADE	sombra					
+Brid-2022-324-106	106	year	1226	YEAR	año					
+Brid-2022-324-107	107	summer	1342	SUMMER	verano					
+Brid-2022-324-108	108	Prosopis alba				5358528	Prosopis alba Griseb.	Plantae	Magnoliopsida	Prosopis
+Brid-2022-324-109	109	lightning	1654	LIGHTNING	relámpago					
+Brid-2022-324-110	110	fight (vb)	1423	FIGHT	pelear					
+Brid-2022-324-111	111	kill	1417	KILL	matar					
+Brid-2022-324-112	112	guitara	3090	GUITAR	guitarra					
+Brid-2022-324-113	113	horse (equine)	615	HORSE	caballo					
+Brid-2022-324-114	114	tail	1220	TAIL	cola					
+Brid-2022-324-115	115	sheep	1331	SHEEP	oveja					
+Brid-2022-324-116	116	goat	1502	GOAT	cabra					
+Brid-2022-324-117	117	he-goat	2475	MALE GOAT	chivo, macho cabrío					
+Brid-2022-324-118	118	ant	587	ANT	hormiga					
+Brid-2022-324-119	119	Rhea americana				2495165	Rhea americana (Linnaeus, 1758)	Animalia	Aves	Rhea
+Brid-2022-324-120	120	leg	1297	LEG	pierna, pata					
+Brid-2022-324-121	121	footprint	318	FOOTPRINT	huella					
+Brid-2022-324-122	122	altar	853	ALTAR	altar					
+Brid-2022-324-123	123	burn (vb intrans)	1428	BURNING	arder					
+Brid-2022-324-124	124	bed	1663	BED	cama					
+Brid-2022-324-125	125	sleep	1585	SLEEP	dormir					
+Brid-2022-324-126	126	lie down	215	LIE DOWN	acostarse, echarse					
+Brid-2022-324-127	127	cousin	1643	COUSIN	primo/prima					
+Brid-2022-324-128	128	flea	232	FLEA	pulga					
+Brid-2022-324-129	129	dog	2009	DOG	perro					
+Brid-2022-324-130	130	louse	1392	LOUSE	piojo					
+Brid-2022-324-131	131	insect	620	INSECT	insecto					
+Brid-2022-324-132	132	float	1574	FLOAT	flotar					
+Brid-2022-324-133	133	fly (vb)	1441	FLY (MOVE THROUGH AIR)	volar					
+Brid-2022-324-134	134	itch	148	ITCH	comezón					
+Brid-2022-324-135	135	two	1498	TWO	dos					
+Brid-2022-324-136	136	little (quantity), few	1242	FEW	poco					
+Brid-2022-324-137	137	potter	359	POTTER	alfarero					
+Brid-2022-324-138	138	hear	1408	HEAR	oír					
+Brid-2022-324-139	139	earth (ground, soil)	1228	EARTH (SOIL)	tierra, suelo					
+Brid-2022-324-140	140	where	1237	WHERE	dónde					
+Brid-2022-324-141	141	how	1239	HOW	cómo					
+Brid-2022-324-142	142	sap	345	SAP	savia					
+Brid-2022-324-143	143	plant (vb)	1486	PLANT (SOMETHING)	plantar, sembrar					
+Brid-2022-324-144	144	mute	2320	MUTE PERSON	mudo					
```

Comment:
Are we sure this refers to the person, not the adjective? If in doubt, we can ask Nicolás directly.

Reply:
The thing is, in all cases it's not an adjective, but rather a construction meaning sth like "languageless" (like in Guaraní) or "s/he who cannot speak", so I'm inclined to think it in reference to the person.